### PR TITLE
Polish the doc of cross_entropy

### LINKED ISSUE #7018: Polish the doc of cross_entropy_op

### PR DESCRIPTION
**Pull request by kuke**
Resolve #7018

### CODE REVIEW

**Comment by kuke** [comment]
HTML view

File: python/paddle/v2/fluid/layers/nn.py [hunk 1]
```diff
@@ -358,7 +359,59 @@ def cos_sim(X, Y, **kwargs):
...
+
+    2) Soft-label cross-entropy:
+	`soft_label = true`, `Label[i, j]` indicates the soft label of class j
+	for sample i:
+
+        .. math::
+
+            Y[i] = \sum_j{-Label[i, j] * log(X[i, j])}
+
+       Please make sure that in this case the summuation of each row of `label`
```
Comment (luotao1) [inline python/paddle/v2/fluid/layers/nn.py]: summation 笔误

File: python/paddle/v2/fluid/layers/nn.py [hunk 2]
```diff
@@ -358,7 +359,59 @@ def cos_sim(X, Y, **kwargs):
...
+       equals one.
+
+    3) One-hot cross-entropy with vecterized `label`:
+	 As a special case of 2), when each row of 'label' has only one
+	 non-zero element (equals 1), soft-label cross-entropy degenerates to a
```
Comment (luotao1) [inline python/paddle/v2/fluid/layers/nn.py]: equals 1 -> the value of this element equals 1?

File: python/paddle/v2/fluid/layers/nn.py [hunk 3]
```diff
@@ -358,7 +359,59 @@ def cos_sim(X, Y, **kwargs):
...
+	 one-hot cross-entropy with one-hot label representation.
+    
+    Args:
+        input (Variable|list):  a 2-D tensor with shape N x D, where N is the 
+            batch size and D is the number of classes. This input is a probability 
+            computed by the previous operator, which is almost always the result
```
Comment (luotao1) [inline python/paddle/v2/fluid/layers/nn.py]: almost 和 always两个选一个

File: python/paddle/v2/fluid/layers/nn.py [hunk 4]
```diff
@@ -358,7 +359,59 @@ def cos_sim(X, Y, **kwargs):
...
+            of a softmax operator.
+        label (Variable|list): the ground truth which is a 2-D tensor. When 
+              `soft_label` is set to `false`, `label` is a tensor<int64> with shape 
+              [N x 1]. When `soft_label` is set to `true`, `label` is a 
+              tensor<float/double> with shape [N x K].
```
Comment (luotao1) [inline python/paddle/v2/fluid/layers/nn.py]: - 391行是N X D，没有用[]，397,398行是不是都统一下
- 这里的N和K是不是也要说明下

File: python/paddle/v2/fluid/layers/nn.py [hunk 5]
```diff
@@ -358,7 +359,59 @@ def cos_sim(X, Y, **kwargs):
...
+        soft_label (bool, via `**kwargs`): a flag indicating whether to interpretate
+              the given labels as soft labels, default `false`.
+
+    Returns:
+         A 2-D tensor with shape [N x 1], the cross entropy loss.
+
+    Raises:
+        `ValueError`: 1) If the 1st dimension of `input` and `label` are not equal; 2) If \ 
+              `soft_label == true`, and the 2nd dimension of `input` and `label` are not \
+               equal; 3) If `soft_label == false`, and the 2nd dimension of `label` is not 1.
```
Comment (luotao1) [inline python/paddle/v2/fluid/layers/nn.py]: 1) the 1st dimension，去掉if
2) when  `soft_label == true`, the 2nd dimension of `input` and `label` are not equal;
3) when `soft_label == false`, the 2nd dimension of `label` is not 1.

**Pull request merged**